src/garage/experiment/snapshotter.py

```diff
@@ -1,127 +1,125 @@
-"""Allows the taking of snapshots."""
+import collections
 import errno
 import os
-from os import listdir
-from os import path as osp
 import pickle
 import re
 
 import joblib
 
-from garage.misc.console import mkdir_p
+from garage.experiment.experiment import ensure_dir
```

Comment:
See https://docs.python.org/3.5/library/pathlib.html#pathlib.Path.mkdir instead (and delete the one from misc.console)